contents.json

```diff
@@ -17624,6 +17624,26 @@
       "date_added": "Apr 8 2019",
       "suggested_by": "@vpeschenkov",
       "stars": 203
+    },
+    {
+      "title": "GitHubSearch",
+      "category-ids": [
+        "github", 
+        "realm", 
```

Comment:
Can you remove realm and developer, let’s put this in a single category. Thx

Reply:
Done